### ISSUE TITLE
suffixList.isRegistrable("bad.bad") returns true

### ISSUE THREAD

**Issue by kamal2311** [whois-server-list/public-suffix-list#9]
suffixList.isRegistrable("bad.bad") returns true -> though bad is not in the suffix list
suffixList.isRegistrable("co m") returns true -> though co m is invalid altogether.

Can you please check?

**Comment by malkusch** [comment]
> bad is not in the suffix list

It is implicit. Check the [PSL algorithm](https://publicsuffix.org/list/):

```
If no rules match, the prevailing rule is "*".
```

> co m is invalid altogether.

Yeah, this is a similar issue like #8.

**Comment by idelpivnitskiy** [comment]
@malkusch Yes, according to the PSL algorithm it's correct. But one of the most useful features is to check is the TLD in the PSL or not. What about introducing a separate method or param, which will exclude wildcard from the rules?
